owlcms/src/main/java/app/owlcms/nui/results/ResultsNavigationContent.java

```diff
@@ -88,9 +89,10 @@ public ResultsNavigationContent() {
 		        "/templates/medalSchedule",
 		        // template name used only to generate the results file name. Localized template determined by
 		        // JXLSTimingStats
-		        "MedalSchedule.xlsx",
+		        Competition::getComputedMedalScheduleTemplateFileName,
+				Competition::setMedalScheduleTemplateFileName,
 		        Translator.translate("Results.MedalSchedule"),
-		        fileName -> fileName.endsWith(".xlsx"));
```

Comment:
I'm not sure if this filter was added for a functional reason or if it was just required for the signature that was in use.

Reply:
probably to avoid using an xls template by mistake